commands.go

```diff
@@ -215,6 +215,7 @@ func displayVersion(output io.Writer, _ *config.Config, flags commandLineFlags,
 
 func displayProfiles(output io.Writer, configuration *config.Config) {
 	profiles := configuration.GetProfiles()
+	configuration.DisplayConfigurationIssues()
```

Comment:
Had added it to get an overview of all issues in all profiles. But I agree that one still needs to look for the profile causing the issue manually.
Without this only issues of the current profile will be displayed but that might be sufficient. Should I change it?

Reply:
Yeah, I think it's fine to show issues in the current profile only. Also if you have inherited profiles it could be more complicated to find the right profile name.

Reply:
Ok